.swiftlint.yml

```diff
@@ -75,11 +72,9 @@ reporter: "xcode" # reporter type (xcode, json, csv, checkstyle, junit)
 opt_in_rules:
   - explicit_init
   - fatal_error_message
-  - number_separator
   - overridden_super_call
   - prohibited_super_call
   - redundant_void_return
-  - sorted_imports
```

Comment:
I'm trashing this rule as well if there are no strong objections. In practice it's pretty annoying to deal with.